upgrade/helpers/tasks.py

```diff
@@ -169,19 +169,18 @@ def sync_capsule_repos_to_upgrade(capsules):
                 capsule_tools = 'Capsule Tools Product'
                 capsule_tools_repo = 'Capsule Tools Repo'
                 hammer_product_create(capsule_tools, '1')
+                tools_label = None
```

Comment:
??

Reply:
Initialization is required, else it fails.